### ISSUE TITLE
Allow sorting tagged photo's

### ISSUE THREAD

**Issue by DrumsnChocolate** [csvalpha/amber-ui#869]
I think sorting direction of a user's tagged photog's should be modifiable. Ascending or descending

**Comment by lodewiges** [comment]
sorting based on what? when they were tagged, how old the photo is? or something else entirely

**Comment by DrumsnChocolate** [comment]
Yes, by date. Ordering by date is current behaviour, but we have no control over the direction

**Comment by jegamboafuentes** [comment]
Hey @DrumsnChocolate 👋

I saw your awesome issue and thought I'd let you know about [Lightning Bounties](https://www.lightningbounties.com/)! 🚀 It's a super easy-to-use app where you can add a Bitcoin reward to this issue, encouraging devs to jump in and help you solve it.

Here’s a quick tutorial to get started: [Lightning Bounties Tutorial](https://docs.lightningbounties.com/docs/posting-a-bounty/create-a-github-issue-and-submit-a-new-reward). 💡 You can post your issue there, set up a bounty, and whoever cracks the code gets rewarded instantly in Bitcoin. 🔥

Let me know if you have any questions or need help setting it up!

Cheers! ⚡

**Comment by lodewiges** [comment]
I am not sure what you want: only the ability to sort or also look up an album name and only see the photos of that album

I think the second would be easier to implement dude to in not requiring a new component

**Comment by DrumsnChocolate** [comment]
We don't need a new component, I'm talking about sorting the tagged photo's you can view via a user's profile

**Comment by lodewiges** [comment]
okay but we need a button for that I think and we don't have a sort button in any other place on the site because in most places we use a filterbar
